ui/app/components/clients/page/counts.hbs

```diff
@@ -58,47 +57,49 @@
       </Hds::Alert>
     {{/if}}
 
-    <div class="is-subtitle-gray has-bottom-margin-m">
-      FILTERS
-      <Toolbar data-test-clients-filter-bar>
-        <ToolbarFilters>
-          <CalendarWidget
-            @startTimestamp={{this.startDate}}
-            @endTimestamp={{this.endDate}}
-            @selectMonth={{this.onDateChange}}
-          />
-          {{#if (or @namespace this.namespaces)}}
-            <SearchSelect
-              @id="namespace-search-select"
-              @options={{this.namespaces}}
-              @inputValue={{if @namespace (array @namespace)}}
-              @selectLimit="1"
-              @disallowNewItems={{true}}
-              @fallbackComponent="input-search"
-              @onChange={{fn this.setFilterValue "ns"}}
-              @placeholder="Filter by namespace"
-              @displayInherit={{true}}
-              class="is-marginless"
-              data-test-counts-namespaces
+    {{#if @startTimestamp}}
```

Comment:
Users who don't have access to the `sys/counters/config` won't have a `@startTimestamp` arg on init, so we don't want to show the filter toolbar, also notice the NaN query param 
<img width="1456" alt="Screenshot 2024-01-25 at 8 05 46 PM" src="https://github.com/hashicorp/vault/assets/68122737/0de898b6-23bc-4bec-ade9-98feeed060d4">

Test policy
```js
path "sys/internal/counters/*" {
    capabilities = ["read"]
}
path "sys/internal/counters/config" {
    capabilities = ["deny"]
}